xml/cap_admin_upgrade.xml

```diff
@@ -131,51 +131,596 @@
+    <para>
+     The <literal>mysql</literal> roles are in high availability mode by setting
+     <literal>config.HA</literal> to <literal>true</literal>, see
+     <xref linkend="sec-cap-upgrade-config-ha"/>.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     The <literal>mysql</literal> roles are in high availability mode by
+     configuring custom sizing values, see
+     <xref linkend="sec-cap-upgrade-custom-sizing"/>.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     The <literal>mysql</literal> roles are in single availability mode, see
+     <xref linkend="sec-cap-upgrade-no-ha"/>.
+    </para>
+   </listitem>
+  </itemizedlist>
+
+  <sect2 xml:id="sec-cap-upgrade-config-ha">
+   <title>&ha; <literal>mysql</literal> through <literal>config.HA</literal></title>
+   <para>
+    This section describes the upgrade process for deployments where the
+    <literal>mysql</literal> roles of <literal>uaa</literal> and
+    <literal>scf</literal> are currently in high availability mode through
+    setting <literal>config.HA</literal> to <literal>true</literal> (see
+    <xref linkend="sec-cap-ha-simple"/>).
+   </para>
+   <para>
+    The following example assumes your current
+    <filename>scf-config-values.yaml</filename> file contains:
+   </para>
+<screen>config:
+  HA: true
+</screen>
+   <procedure>
+    <step>
+     <para>
+      For the <literal>uaa</literal> deployment, scale the instance count of the
+      <literal>mysql</literal> role to 1 and remove the associated persistent
+      volume claim.
+     </para>
+     <substeps>
+      <step>
+       <para>
+        Scale the <literal>mysql</literal> instance count to 1.
+       </para>
+       <para>
+	Create a custom configuration file, called
+	<filename>ha-strict-false-single-mysql.yaml</filename> in this example,
+	with the values below. The file uses the
+	<literal>config.HA_strict</literal> feature to allow setting the
+	<literal>mysql</literal> role to a single instance while other roles are
+	kept HA. Note that it is assumed <literal>config.HA=true</literal> is set
+	in your existing <filename>scf-config-values.yaml</filename> file.
+       </para>
+<screen>config:
+  HA_strict: false
+sizing:
+  mysql:
+    count: 1
+</screen>
+       <para>
+        Perform the scaling.
+       </para>
+<screen>&prompt.user;helm upgrade susecf-uaa suse/uaa \
+--values scf-config-values.yaml \
+--values <replaceable>ha-strict-false-single-mysql.yaml</replaceable> \
+--version 2.18.0
+</screen>
+       <para>
+        Monitor progress using the <command>watch</command> command.
+       </para>
+       &watch-uaa;
+      </step>
+      <step>
+       <para>
+        Delete the persistent volume claim (PVC) for <literal>mysql-1</literal>
+	from the <literal>uaa</literal> namespace. <emphasis>Do not</emphasis>
+	delete the persistent volume (PV).
+       </para>
+<screen>&prompt.user;kubectl delete persistentvolumeclaim --namespace uaa mysql-data-mysql-1</screen>
```

Comment:
This assumes there were only 2 instances of mysql to begin with? I think `config.HA` setting defaults the mysql count to 3.

Reply:
Update wording for this and other similar occurrences in https://github.com/SUSE/doc-cap/pull/650/commits/bb8821002c60a20ac3e176003e9367c5c9f6f49c